package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "add-variable-declarations",
+  "version": "0.0.0-development",
   "description": "Add variable declarations for global assignments.",
-  "main": "dist/add-variable-declarations.js",
-  "jsnext:main": "dist/add-variable-declarations.mjs",
+  "main": "src/index.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "build": "rollup -c",
     "test": "mocha",
-    "prepublish": "npm run build",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "prepublish": "tsc --project ."
```

Comment:
Re-added a `build` script and documented how to develop using it and using the `--watch` option.